packages/netlify-cms-backend-github/src/implementation.js

```diff
@@ -150,49 +150,32 @@ export default class GitHub {
   }
 
   unpublishedEntries() {
-    return this.api
-      .listUnpublishedBranches()
-      .then(branches => {
-        const sem = semaphore(MAX_CONCURRENT_DOWNLOADS);
-        const promises = [];
-        branches.map(branch => {
-          promises.push(
-            new Promise(resolve => {
-              const slug = branch.ref.split('refs/heads/cms/').pop();
-              return sem.take(() =>
-                this.api
-                  .readUnpublishedBranchFile(slug)
-                  .then(data => {
-                    if (data === null || data === undefined) {
-                      resolve(null);
-                      sem.leave();
-                    } else {
-                      const path = data.metaData.objects.entry.path;
-                      resolve({
-                        slug,
-                        file: { path },
-                        data: data.fileData,
-                        metaData: data.metaData,
-                        isModification: data.isModification,
-                      });
-                      sem.leave();
-                    }
-                  })
-                  .catch(() => {
-                    sem.leave();
-                    resolve(null);
-                  }),
-              );
-            }),
-          );
-        });
-        return Promise.all(promises);
-      })
-      .catch(error => {
-        if (error.message === 'Not Found') {
-          return Promise.resolve([]);
-        }
-        return error;
+    return this.api.listUnpublishedBranches().then((branches) => {
+      const sem = semaphore(MAX_CONCURRENT_DOWNLOADS);
+      const promises = [];
+      branches.map((branch) => {
+        promises.push(new Promise((resolve, reject) => {
+          const slug = branch.ref.split(`refs/heads/${this.api.cms_branch_prefix}`).pop();
+          return sem.take(() => this.api.readUnpublishedBranchFile(slug).then((data) => {
+            if (data === null || data === undefined) {
+              resolve(null);
+              sem.leave();
+            } else {
+              const path = data.metaData.objects.entry.path;
+              resolve({
+                slug,
+                file: { path },
+                data: data.fileData,
+                metaData: data.metaData,
+                isModification: data.isModification,
+              });
+              sem.leave();
+            }
+          }).catch((err) => {
+            sem.leave();
+            resolve(null);
+          }));
+        }));
```

Comment:
It looks like something's been truncated here, since this is a syntax error and the function no longer returns anything, and the PR as a whole needs to be formatted by running `yarn run format:prettier --write`.

Reply:
Checking. This was a merge conflict in the branch which I tried to resolve on github. Bad idea.